hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/models/UpdateTrackingAccount.java

```diff
@@ -166,6 +169,12 @@ public Bytes getCode() {
         return updatedCode == null ? account.getCode() : updatedCode;
     }
 
+    @Override
+    public void setCode(Bytes code) {
+        this.updatedCode = code;
+        this.updatedCodeHash = null;
+    }
+
```

Comment:
(Leaving comment here because I can't leave it where it belongs at L225:)

If `getOriginalStorageValue` is supposed to "Retrieves the original value from before the current transaction in the account storage given its key" (according to its Javadoc in `
org.hyperledger.besu.evm.account.AccountState`) then I don't understand why it is conditional on the flag `storageWasCleared`.